nautobot/extras/utils.py

```diff
@@ -880,3 +883,30 @@ def bulk_delete_with_bulk_change_logging(qs, batch_size=1000):
         finally:
             change_context.defer_object_changes = False
             change_context.reset_deferred_object_changes()
+
+
+def format_query_param_dict(param_dict, view_name, non_filter_params):
```

Comment:
I feel like this is a really generic name - can we come up with something clearer, maybe like `fixup_filterset_query_params`? I dunno.